### PR TITLE
STM32 USB Serial scope

### PR DESCRIPTION
**Pull request by arpruss**
Thanks!

### CODE REVIEW

File: 1209/AAAA/index.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: STM32 USB Serial Oscilloscope
+owner: Omega Centauri Software
```
Comment (Arachnid) [inline 1209/AAAA/index.md]: This needs to match the name of the organisation folder.

Comment (arpruss) [reply 1209/AAAA/index.md]: 1. There is no PCB design. If all one wants is to monitor TTL stuff, it should eventually work with a standard blue or black pill STM32F103 board with an F303 soldered on it.

2. Do you mean that I should remove the spaces from the owner name, to match the OmegaCentauriSoftware folder?

Comment (Arachnid) [reply 1209/AAAA/index.md]: Yes, the 'owner' field needs to be the directory name of your organisation.

Comment (arpruss) [reply 1209/AAAA/index.md]: OK, fixed.

**Comment by Arachnid** [comment]
Please open a new PR when your repository has hardware and software designs and an open-source license.

**Pull request closed without merging**